js/script.js

```diff
@@ -50,6 +51,9 @@ const addTask = () => {
 const createTaskCard = (task) => {
   const taskCard = document.createElement("div");
   taskCard.className = "task-card";
+  if (task.isCompleted) {
+    taskCard.classList.add("isCompleted");
```

Comment:
```suggestion
    taskCard.classList.add("task-completed");
```

`class` name should not be as like as` js variables`. It should not have any `camelCase`.